lib/google_maps_geocoder.rb

```diff
@@ -11,6 +11,8 @@ class GoogleMapsGeocoder
   # Self-explanatory
   attr_reader :city, :country_long_name, :country_short_name, :county, :lat, :lng, :postal_code, :state_long_name, :state_short_name
 
+  GOOGLE_API_URI = 'https://maps.googleapis.com/maps/api/geocode/json'
```

Comment:
I've moved it here only for readability reason, but if you are agains this new constant in the class, I can move it back